app/models/comment.rb

```diff
@@ -0,0 +1,9 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+  after_save :update_comment_counter
+
+  def update_comment_counter
+    post.update(comments_counter: post.comments.all.length)
+  end
```

Comment:
- [ ] [OPTIONAL] Please consider using the `increment` method I talked about earlier in order to make your code more readable and concise 👍 
``` ruby
def update_comment_counter
  post.increment!(comments_counter)
end
```